### PR TITLE
[MBL-17481][Parent] Project init

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: in the ticket

refs: MBL-17481
affects: Parent
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

File: apps/parent/build.gradle [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        viewBinding true
+        dataBinding true
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/parent/build.gradle:63]: I used viewBinding at some places, removed dataBinding

File: apps/parent/build.gradle [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+android {
+    namespace 'com.instructure.parentapp'
+    compileSdk Versions.COMPILE_SDK
+
+    defaultConfig {
+        applicationId "com.instructure.parentapp"
+        minSdkVersion Versions.MIN_SDK
+        targetSdkVersion Versions.TARGET_SDK
+        versionCode 50
+        versionName "3.10.0"
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/parent/build.gradle:41]: Yeah we'll find it out before release

File: apps/parent/src/main/java/com/instructure/parentapp/features/inbox/list/ParentInboxRepository.kt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+class ParentInboxRepository(
+    inboxApi: InboxApi.InboxInterface,
+    private val coursesApi: CourseAPI.CoursesInterface,
+    groupsApi: GroupAPI.GroupInterface,
+    progressApi: ProgressAPI.ProgressInterface
+) : InboxRepository(inboxApi, groupsApi, progressApi) {
+
+    override suspend fun getCourses(params: RestParams): DataResult<List<Course>> {
+        val coursesResult = coursesApi.getFirstPageCourses(params)
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/parent/src/main/java/com/instructure/parentapp/features/inbox/list/ParentInboxRepository.kt:40]: Will be done in this ticket: MBL-17609

File: apps/parent/src/main/java/com/instructure/parentapp/features/inbox/list/ParentInboxRepository.kt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import com.instructure.canvasapi2.apis.ProgressAPI
+import com.instructure.canvasapi2.builders.RestParams
+import com.instructure.canvasapi2.models.Course
+import com.instructure.canvasapi2.utils.DataResult
+import com.instructure.canvasapi2.utils.depaginate
+import com.instructure.canvasapi2.utils.hasActiveEnrollment
+import com.instructure.canvasapi2.utils.isValidTerm
+import com.instructure.pandautils.features.inbox.list.InboxRepository
+
+class ParentInboxRepository(
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/parent/src/main/java/com/instructure/parentapp/features/inbox/list/ParentInboxRepository.kt:32]: Will be done in the ingegration ticket

File: apps/parent/src/main/java/com/instructure/parentapp/features/help/ParentHelpLinkFilter.kt [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ */
+
+package com.instructure.parentapp.features.help
+
+import com.instructure.canvasapi2.models.Course
+import com.instructure.canvasapi2.models.HelpLink
+import com.instructure.pandautils.features.help.HelpLinkFilter
+
+class ParentHelpLinkFilter : HelpLinkFilter {
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/parent/src/main/java/com/instructure/parentapp/features/help/ParentHelpLinkFilter.kt:24]: Will be done here: MBL-17610

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by hermannakos — APPROVED** [review]

**Pull request merged**